docs/install/command-line.md

```diff
@@ -1,52 +1,102 @@
+This guide describes the available installation processes for IPFS Kubo, a Go-based implementation of the InterPlanetary File System (IPFS) protocol. Kubo was the first implementation of IPFS, and is the most widely used implementation today. Kubo allows you to do everything that IPFS Desktop can do, but at a more granular level, since you can specify which commands to run. Kubo has the following features:
 
-![A terminal window running the IPFS daemon in Ubuntu.](./images/command-line/wsl-running-ipfs-in-linux.png)
+- An IPFS daemon server
+- Extensive command line tooling
+- An HTTP RPC API for controlling the node
+- An HTTP Gateway for serving content to HTTP browsers
+- Binaries for Windows, MacOS, Linux, FreeBSD and OpenBSD
+
+Installing Kubo in the command line is handy for many use cases, such as building applications and services on top of an IPFS node, or setting up a node without a user interface (which is usually the case with remote servers or virtual machines).  
+
+To get started, familiarize yourself with the system requirements. Then, determine if you'd like to install Kubo using one of the 5 official binary distributions, or build Kubo from source. Once you've installed Kubo, determine which node to use in the command line. Finally, check out the next steps.
+
+:::warning
+Building from source is only recommended if you are running Kubo on a system with severe resource constraints, or are contributing to the Kubo project. 
+:::
 
 ## System requirements
 
-Kubo IPFS requires 512MiB of memory and can run an IPFS node on a Raspberry Pi. However, how much disk space your IPFS installation takes up depends on how much data you're sharing. A base installation takes up about 12MB of disk space. One can enable automatic garbage collection via [--enable-gc](/reference/kubo/cli.md#ipfs-daemon) and adjust the [default maximum disk storage](https://github.com/ipfs/kubo/blob/v0.17.0/docs/config.md#datastorestoragemax) for data retrieved from other peers.
+Kubo runs on most Windows, MacOS, Linux, FreeBSD and OpenBSD systems that meet the following requirements:
+
+- 6 GiB of memory.
+- 2 CPU cores (kubo is highly parallel).
+
+Note the following:
+- The amount of disk space your IPFS installation uses depends on how much data you're sharing. A base installation uses around 12MB of disk space.
+- You can enable automatic garbage collection via [--enable-gc](../reference/kubo/cli.md#ipfs-daemon) and adjust using [default maximum disk storage](https://github.com/ipfs/kubo/blob/v0.17.0/docs/config.md#datastorestoragemax) for data retrieved from other peers.
+
+### Kubo on resource-contrained systems 
```

Comment:
Typo: contrained -> constrained

Reply:
🦅 👁️ 

Also, weird cus Vale didn't flag it when I ran it locally. I'm wondering if it was the `-`.  Anyways, weird, I'm making a note of it for the linter project